resources/crds/ripsaw_v1alpha1_vdbench_cr.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: vdbench-benchmark
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+   server: "marquez.perf.lab.eng.rdu2.redhat.com"
+    port: 9200
+  metadata_collection: true
+  metadata_targeted: false
+  clustername: myk8scluster
+  test_user: ripsaw
+  workload:
+    name: "vdbench"
+    args:
+      servers: 1
+      #pin_server: 'vdbench'
+      threads: 1
+      fileselect: random
+      fileio: random
+      jobs:
+        - name: RandomRead
+          op: read
+        - name: RandomWrite
+          op: write
+       # - name: RandomMix75
+       #   op: read
+       #   rdpct: 75
+      iorate:
```

Comment:
add comment explaining what the unit is, is it requests/sec/pod or requests/sec/thread?